libc/config/uefi/entrypoints.txt

```diff
@@ -0,0 +1,473 @@
+set(TARGET_LIBC_ENTRYPOINTS
+    libc.src.compiler.__stack_chk_fail
+
+    # ctype.h entrypoints
```

Comment:
Do these all pass tests? It's probably best to start with a really minimal set and then add the entrypoints / headers as we get support in a follow-up

Reply:
Tests don't quite build from what I can tell

Reply:
Ok, tests build now and run! To run tests, run `ninja check-libc` and let it fail executing on the host. Next, cd into the directory containing a set (`include` or `src` under `libc/test`). Collect the tests like this:
```
find . -type f -name 'libc.test.*__hermetic__.__build__' | tr '/' '\\' > startup.nsh
```
Then pass the directory you are in to QEMU, example:
```
qemu-system-aarch64 -machine virt,gic-version=max,accel=kvm:tcg -cpu max -drive if=pflash,format=raw,unit=0,readonly=on,file=/nix/store/i3d83jqxl2w85p720f88ymdxwlpaz9vf-OVMF-202408.01-fd/FV/AAVMF_CODE.fd -serial stdio -m 4G -drive file=fat:rw:/home/ross/llvm-project/runtimes/build/libc/test/include,format=raw,media=disk
```

Now all tests will run.

Reply:
We now have a test driver in `libc/testing/scripts/uefi_runner.py`. To use it, install OVMF for the system you're compiling for. Arch Linux has `edk2-ovmf` and NixOS has `pkgs.OVMF`. Just set `CMAKE_CROSSCOMPILING_EMULATOR` to the script. If OVMF is not at the path the script expects, use `$OVMF_PATH` to set it to the directory it exists at.